apps/contracts/contracts/Zengo.sol

```diff
@@ -102,7 +102,8 @@ contract ZengoDAO is Constants, ZengoStorage, GStates, PermissionsEnumerable, Co
         string memory _proposalType,
         string memory _streetAddress,
         uint256 _latitude,
-        uint256 _longitude
+        uint256 _longitude,
+        uint256 _timestamp
```

Comment:
Can we call this value with a more representative name?
It is not mandatory to change it now, but if it can be considered it would be great.

Context:

- If the proposer is uploading an image, this `_timestamp` is **the date the image was taken**. For this escenario something like "evidence_date" or "date_of_evidence", I don't know, could work better and could avoid confusion, especially thinking ahead.

cc: @shyam-patel-kira @HabacucMX

Reply:
pushed an update